### PR TITLE
[Author] [New] Add embedo@0.3.6 w/ git auto-update

### PR DESCRIPTION
**Pull request by shobhitsharma**
Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/shobhitsharma/embedo
 * Official website (optional, not the repository): https://shobhitsharma.github.io/embedo/
 * NPM package url (optional): https://www.npmjs.com/package/embedo
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 3*
   - Count of stars: 21*
   - Count of forks: 1*
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by shobhitsharma** [comment]
@PeterBot Missed a `,`. Fixed now.

File: ajax/libs/embedo/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "author": "Shobhit Sharma <hi@shobh.it>",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/makeusabrew/bootbox.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "embedo.js",
+          "embedo.min.js"
```
Comment (PeterDaveHello) [inline ajax/libs/embedo/package.json]: What about `embedo*`?

**Comment by PeterDaveHello** [comment]
@shobhitsharma is it possible for you to squash the commits locally? Just asking, no pressure.

File: ajax/libs/embedo/package.json (hunk 1 above)
Comment (shobhitsharma) [inline ajax/libs/embedo/package.json]: Sure. 1 second.

**Comment by shobhitsharma** [comment]
@PeterDaveHello Done.

**Comment by PeterDaveHello** [comment]
@shobhitsharma looks not, there seems to be more commits, you wanna to take a look?

**Comment by PeterDaveHello** [comment]
To prevent resource wasting, I manually cancelled some of the previous CI tests.

**Comment by PeterDaveHello** [comment]
@shobhitsharma I guess you are using GitHub web gui not a local git tool? Then you can't squash them into single commit.

File: ajax/libs/embedo/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "description": "A simple, fast, lightweight and standalone social media content embed plugin for web",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/shobhitsharma/embedo.git"
+  },
+  "homepage": "https://shobhitsharma.github.io/embedo/",
+  "author": "Shobhit Sharma <hi@shobh.it>",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/makeusabrew/bootbox.git",
```
Comment (PeterDaveHello) [inline ajax/libs/embedo/package.json]: Is this the correct target? A totally different repo?

Comment (shobhitsharma) [inline ajax/libs/embedo/package.json]: Ah, went along with merge. Sorted now. All looks good to me.

File: ajax/libs/embedo/3.6.0/embedo.js [hunk 3]
```diff
@@ -0,0 +1,978 @@
+/**
```
Comment (hare1039) [inline ajax/libs/embedo/3.6.0/embedo.js]: Please remove the `3.6.0` folder.

**Comment by hare1039** [comment]
Also, auto-update can get `0.3.3` correctly, so please remove that folder too.

**Comment by hare1039** [comment]
My diff reports some difference between `embedo.min.js` from your repository and this PR.
Would you mind to upload this file again?

**Comment by shobhitsharma** [comment]
Since I'm using web interface, its not possible to remove directory. I'll snd another pull. Closing this.

**Comment by PeterDaveHello** [comment]
@shobhitsharma I think you can simply remove the files :) Closing this PR, removing the files, and then creating PR could be more effective :+1:

**Comment by shobhitsharma** [comment]
@PeterDaveHello @hare1039 Cool. didnt know directories remove automatically, I removed all files and uploaded  fresh ones. Let me know.

**Comment by PeterDaveHello** [comment]
@sufuf3 please help squash these commits if it looks good to you.

**Comment by hare1039** [comment]
ping @sufuf3 @extend1994

**Comment by PeterDaveHello** [comment]
@maruilian11 please help review this PR, and if it looks go to you, please also help improve the commit message, rebase the branch, and assign to me, thanks.

**Comment by maruilian11** [comment]
Hi @shobhitsharma I think this PR is good, but I found there is v0.1.0 on npm package, not on git repo.
Would you mind if I add a commit following with your commit to add v0.1.0? Thanks.

**Comment by shobhitsharma** [comment]
@maruilian11 Sure. Please go ahead.

**Pull request merged**